pkg/config/config.go

```diff
@@ -88,5 +92,53 @@ func LoadFromFile(file string) (Config, error) {
 		return cfg, err
 	}
 	err = yaml.UnmarshalStrict(data, &cfg)
-	return cfg, err
+	if err != nil {
+		return Config{}, err
+	}
+	return cfg, nil
+}
+
+//Load will load the config files making sure that the options override are correct
+func (c Config) Load() (Config, error) {
```

Comment:
suggestion(non-blocking): add to the doc comment that the returned Config should be used in place of the original, since the original will go unmodified (due to the value receiver)

Reply:
good find, I'll use a pointer to avoid this possible confusion